src/main/java/somethingrandom/interfaceadapters/additem/AddItemPresenter.java

```diff
@@ -20,4 +20,9 @@ public void prepareSuccessView(String response){
         AddItemState addItemState = addItemViewModel.getState();
     }
 
+    @Override
+    public void prepareFailView() {
+
```

Comment:
Filling this in would be _good_, but at this point I'd rather have this merged and worry about it later.